<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseWindow</class>
 <widget class="QMainWindow" name="ChooseWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>难度选择</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1600</width>
      <height>1600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pictures.qrc">:/new/prefix1/pic/choosebgpic.png</pixmap>
    </property>
   </widget>
   <widget class="MyLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>420</y>
      <width>667</width>
      <height>165</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pictures.qrc">:/new/prefix1/pic/difficulty1.png</pixmap>
    </property>
   </widget>
   <widget class="MyLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>620</y>
      <width>667</width>
      <height>165</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pictures.qrc">:/new/prefix1/pic/difficulty2.png</pixmap>
    </property>
   </widget>
   <widget class="MyLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>820</y>
      <width>667</width>
      <height>165</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pictures.qrc">:/new/prefix1/pic/difficulty3.png</pixmap>
    </property>
   </widget>
   <widget class="MyLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>1020</y>
      <width>667</width>
      <height>165</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pictures.qrc">:/new/prefix1/pic/difficulty4.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header>mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="pictures.qrc"/>
 </resources>
 <connections/>
</ui>
